src/main/res/values/strings.xml

```diff
@@ -844,9 +840,9 @@
     <!-- %1$d will be replaced by the number of messages, you can assume plural/lots here. %2$s will be replaced by a timespan option. -->
     <string name="autodel_device_ask">Do you want to delete %1$d messages now and all newly fetched messages \"%2$s\" in the future?\n\n• This includes all media\n\n• Messages will be deleted whether they were seen or not\n\n• \"Saved messages\" will be skipped from local deletion</string>
     <!-- %1$d will be replaced by the number of messages, you can assume plural/lots here. %2$s will be replaced by a timespan option. -->
-    <string name="autodel_server_ask">Do you want to delete %1$d messages now and all newly fetched messages \"%2$s\" in the future?\n\n⚠️ This includes e-mails, media and \"Saved messages\" in all server folders\n\n⚠️ Do not use this function if you want to keep data on the server\n\n⚠️ Do not use this function if you are using other e-mail clients besides Delta Chat</string>
+    <string name="autodel_server_ask">Do you want to delete %1$d messages now and all newly fetched messages \"%2$s\" in the future?\n\n⚠️ This includes messages, media and \"Saved messages\" in all server folders\n\n⚠️ Do not use this function if you want to keep data on the server or if you use other apps beside Delta Chat</string>
```

Comment:
> Do not use this function if you want to keep data on the server or if you use other apps beside Delta Chat

Here I think the meaning becomes unclear by replacing "e-mail clients" with "apps". Like, of course the user is using other apps beside Delta Chat: Signal, a meditation app, their banking app...

Also, I think it is good to explicitly note that the deletion also includes "e-mails", not only (Delta Chat) messages.

This is a string that's only shown for non-chatmail profiles, where users are usually more aware that they are chatting over e-mail. So, I think we could just keep the previous string.

Reply:
fine with staying with "e-mails" instead of "messages".

questionable is the second part:  "other email clients _besides_ Delta Chat implies that Delta Chat is just an email client. again, it is also how we internally talk about that, translator see it etc.

maybe say `Do not use this function if you are using classic e-mail clients as well`

or `Do not use this function if you are using classic e-mail clients on this profile`